From ef7641747883bbc5a5e7bcbce64bdbcd096469fa Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Mon, 15 Jun 2015 10:11:33 +0800
Subject: [PATCH 425/462] DW9718: remove redundant print log

VCM focus position could be a reference while debugging.
Actually the value would be around 300.

Change-Id: I29ddabb5221b6d56e7acbc68967a024647593e87
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-39329
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/60522
Approver: Yunliang Ding <yunliang.ding@intel.com>
Reviewed-by: Yunliang Ding <yunliang.ding@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: IMIN (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c b/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
index 9e04787..394cc67 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
@@ -86,7 +86,7 @@ int dw9718_t_focus_abs(struct v4l2_subdev *sd, s32 value)
 
 	value = clamp(value, 0, DW9718_MAX_FOCUS_POS);
 	ret = dw9718_i2c_wr16(client, DW9718_DATA_M, value);
-	pr_info("%s: value = %d\n", __func__, value);
+	/*pr_info("%s: value = %d\n", __func__, value);*/
 	if (ret < 0)
 		return ret;
 
-- 
1.9.1

